deployment/existing/auth_oidc/authservice.tmpl

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: authservice
+  namespace: kubeflow
+spec:
+  type: ClusterIP
+  selector:
+    app: authservice
+  ports:
+  - port: 8080
+    name: http-authservice
+    targetPort: http-api
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: authservice
+  namespace: kubeflow
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "false"
+      labels:
+        app: authservice
+    spec:
+      containers:
+      - name: authservice
+        image: gcr.io/arrikto/kubeflow/oidc-authservice:v0.1
```

Comment:
Where can we find the source code for this image?

Reply:
Is it https://github.com/yanniszark/ambassador-auth-oidc ?

Reply:
Exactly, in the `kubeflow` branch.
The upstream repo is https://github.com/ajmyyra/ambassador-auth-oidc.
I have made upstream issues for some of the fixes I had to make and I'll try to merge all the extra code upstream soon.

Reply:
is the branch 'kubeflow' or 'feature-kubeflow'?